File: Networking/src/main/java/request/CreateEventEntriesRequest.java
package request;


import example.model.Office;

import java.util.List;

public class CreateEventEntriesRequest implements Request {
    private final List<Office> eventEntries;

    public CreateEventEntriesRequest(List<Office> eventEntries) {
        this.eventEntries = eventEntries;
    }

    public List<Office> getEventEntries() {
        return eventEntries;
    }
}

File: Networking/Request/CreateEventEntriesRequest.cs
using System.Text.Json.Serialization;
using mpp_proiect_csharp_DianaGliga11.Model;

namespace Networking.Request
{
    public class CreateEventEntriesRequest : IRequest
    {
        [JsonPropertyName("eventEntries")]
        public List<Office> EventEntries{get;set;} = null!;

        [JsonConstructor]
        public CreateEventEntriesRequest()
        {
        }

        public CreateEventEntriesRequest(List<Office> eventEntries)
        {
            this.EventEntries = eventEntries;
        }
    }
}